src/commands/settings/settings-karma.ts

```diff
@@ -0,0 +1,34 @@
+import type { ChatInputCommandInteraction } from 'discord.js';
+
+import { getChannelFirestoreReference } from '../../utilities/firestore-helper.js';
+
+/** Enables or disables karma tracking on a channel in Firebase */
+export async function karma(
+  firestore: FirebaseFirestore.Firestore,
+  interaction: ChatInputCommandInteraction<'cached' | 'raw'>
+): Promise<void> {
+  let karmaTracking = interaction.options.getBoolean('value');
+  const channel = interaction.channel;
+  if (!channel) {
+    throw new Error();
```

Comment:
Can we give this error a message? Do we want to throw a RustyBotCommandError instead?

Reply:
In #110 I remove the check entirely since its unnecessary in the reworked command.

Reply:
Ah okay